src/http_server.cc

```diff
@@ -2899,7 +2904,7 @@ HTTPAPIServer::InferRequestClass::FinalizeResponse(
   triton::common::TritonJson::Value response_json(
       triton::common::TritonJson::ValueType::OBJECT);
 
-  const char* request_id;
+  const char* request_id = "";
```

Comment:
Base on the check below (`request_id != nullptr`), should initialize to `nullptr`

Reply:
Thanks, updated.